### PR TITLE
feat: POST->GET - Market API

### LINKED ISSUE #1070: [FEATURE] POST-> GET (Market API)
**Is your feature request related to a problem? Please describe.**

- For APIs related to "Market", the HTTP method of the API for data reference is POST.
- This is not RESTful, there is no particular need for POST, and it seems inappropriate.

**Describe the solution you'd like**

- Change the method to GET.

### PR DESCRIPTION
**Pull request by YoshihitoAso**
close #1070

### CODE REVIEW

File: docs/ibet_wallet_api.yaml [hunk 1]
```diff
@@ -3254,115 +3254,38 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/NotSupportedErrorResponse'
-  /DEX/Market/OrderBook/StraightBond:
```
Comment (YoshihitoAso) [inline docs/ibet_wallet_api.yaml:3257]: This API has been removed.

File: docs/ibet_wallet_api.yaml [hunk 2]
```diff
@@ -3254,115 +3254,38 @@ paths:
...
-            application/json:
-              schema:
-                $ref: '#/components/schemas/RequestValidationErrorResponse'
-        '404':
-          description: Not Found
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/NotSupportedErrorResponse'
-  /DEX/Market/LastPrice/StraightBond:
```
Comment (YoshihitoAso) [inline docs/ibet_wallet_api.yaml:3289]: This API has been removed.

File: docs/ibet_wallet_api.yaml [hunk 3]
```diff
@@ -3677,51 +3594,6 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/NotSupportedErrorResponse'
-  /DEX/OrderList/Share:
```
Comment (YoshihitoAso) [inline docs/ibet_wallet_api.yaml:3680]: This API has been removed.

File: bin/healthcheck_processor_notification.sh [hunk 4]
```diff
@@ -24,26 +24,14 @@ if [ "${TOKEN_NOTIFICATION_ENABLED}" = 1 ]; then
...
-      PROC_LIST="${PROC_LIST} batch/processor_Notifications_Bond_Exchange.py"
-    fi
-  fi
-
-  if [ "${SHARE_TOKEN_ENABLED}" = 1 ]; then
-    if [ ! -z "${IBET_SHARE_EXCHANGE_CONTRACT_ADDRESS}" ]; then
-      PROC_LIST="${PROC_LIST} batch/processor_Notifications_Share_Exchange.py"
-    fi
-  fi
-
```
Comment (YoshihitoAso) [inline bin/healthcheck_processor_notification.sh:38]: These processors have been removed.

File: app/model/schema/dex_market.py [hunk 5]
```diff
@@ -42,11 +42,14 @@ class OrderType(str, Enum):
...
-    token_address: str = Field(description="Token address")
-    order_type: OrderType = Field(description="Order type to be executed by the Orderer. "
+@dataclass
+class ListAllOrderBookQuery:
+    account_address: Optional[str] = Query(
+        default=None,
+        description="Orderer's account address. Orders from the given address will not be included in the response."
+    )
+    token_address: str = Query(description="Token address")
+    order_type: OrderType = Query(description="Order type to be executed by the Orderer. "
```
Comment (YoshihitoAso) [inline app/model/schema/dex_market.py:52]: Changed to query parameters.

File: app/model/schema/node_info.py [hunk 6]
```diff
@@ -44,14 +41,10 @@ class GetNodeInfoResponse(BaseModel):
...
-    ibet_straightbond_exchange_abi: Optional[object]
     ibet_membership_exchange_address: Optional[str]
     ibet_membership_exchange_abi: Optional[object]
     ibet_coupon_exchange_address: Optional[str]
     ibet_coupon_exchange_abi: Optional[object]
-    ibet_share_exchange_address: Optional[str]
-    ibet_share_exchange_abi: Optional[object]
     ibet_escrow_address: Optional[str]
     ibet_escrow_abi: Optional[object]
     ibet_security_token_escrow_address: Optional[str]
```
Comment (YoshihitoAso) [inline app/model/schema/node_info.py:50]: Changed response schema.

**Review by purplesmoke05 — APPROVED** [review]
I think everything looks fine.

**Pull request merged**